app/src/main/res/layout/activity_main.xml

```diff
@@ -77,6 +77,14 @@
             android:checked="true"
             android:text="@string/auto_load_description" />
 
+        <CheckBox
+            android:id="@+id/handle_http_scheme"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="5dp"
+            android:checked="true"
```

Comment:
Since the option is disabled by default, wouldn't it be more logical to set this to `android:checked="false"` ?